src/patternfly/layouts/Flex/flex.scss

```diff
@@ -10,6 +10,10 @@ $pf-l-flex--variable-map: build-variable-map("pf-l-flex--spacer", $pf-l-flex--sp
   --pf-l-flex--m-row--AlignItems: baseline;
   --pf-l-flex--m-row-reverse--AlignItems: baseline;
   --pf-l-flex--item--Order: 0;
+  // stylelint-disable length-zero-no-unit
```

Comment:
looks like you added the unit - do you still need the stylelint rules?

Reply:
Not sure if they're needed or not, but wanted to mention again. Looks like they should be able to be removed?